index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head dir="ltr" lang="es">
+        <meta charset="utf-8">
+        <title>Carrusel de imagenes</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js" defer></script>
+    </head>
+    <body>
+        <div id="content">
+            <section id="arrow_left">
+                <button onclick="changeImageLeft()">←</button>
+            </section>
+            <section>
+                <img id="images" src="assets/image_1.jpg" alt="Carrusel de imágenes">
+            </section>
+            <section id="arrow_right">
+                <button onclick="changeImageRight()">→</button>
+            </section>
```

Comment:
```suggestion
            <div id="arrow_left">
                <button onclick="changeImageLeft()">←</button>
            </div>
            <div>
                <img id="images" src="assets/image_1.jpg" alt="Carrusel de imágenes">
            </div>
            <div id="arrow_right">
                <button onclick="changeImageRight()">→</button>
            </div>
```
Section es para agrupar contenido semánticamente similar, en este caso solo necesita organizar el código, así que un div es la etiqueta adecuada.